styles/typography/typography-preset-1.json

```diff
@@ -0,0 +1,40 @@
+{
+	"version": 3,
+	"$schema": "https://schemas.wp.org/trunk/theme.json",
+	"title": "Beiruti & Literata",
+	"slug": "typography-preset-1",
+	"styles": {
+		"typography": {
+			"fontFamily": "var:preset|font-family|literata",
+			"fontSize": "var:preset|font-size|medium",
+			"fontStyle": "normal",
+			"fontWeight": "300",
+			"letterSpacing": "-0.22px",
+			"lineHeight": "1.5"
+		},
+		"blocks": {
+			"core/site-title": {
+				"typography": {
+					"fontFamily": "var:preset|font-family|beuritu",
+					"fontWeight": "600",
+					"textTransform": "uppercase",
+					"letterSpacing": "2.4px"
+				}
+			},
+			"core/post-title": {
+				"typography": {
+					"fontSize": "var:preset|font-size|xx-large",
```

Comment:
In Figma, the post title on the single post is set to 58px.
However, in the archive template markup, the post title is set to large, 32px. In the single post template markup, the post title is set to x-large, 48px.
Meaning that setting the size to 58px here won't have any affect on this two titles.